core/src/main/java/hudson/FilePath.java

```diff
@@ -335,7 +335,8 @@ public static String normalize(@Nonnull String path) {
                 tokens.add(path.substring(s, i));
                 s = i;
                 // Skip any extra separator chars
-                while (++i < end && ((c = path.charAt(i)) == '/' || c == '\\')) { }
+                //noinspection StatementWithEmptyBody
+                while (++i < end && ((c = path.charAt(i)) == '/' || c == '\\'));
```

Comment:
Easier to read as in https://github.com/jenkinsci/jenkins/pull/4020/files#diff-bf28198e17c0bde7c545a51b280856b9R132

```suggestion
                while (++i < end && ((c = path.charAt(i)) == '/' || c == '\\'))
                    ;
```